cockpit/src/main/java/com/polidea/cockpit/manager/CockpitManager.kt

```diff
@@ -41,9 +45,10 @@ object CockpitManager {
 
     fun clear() {
         params.clear()
+        defaultParams.clear()
```

Comment:
Default params are loaded from an immutable file. There is no need to clear them.
Anyways, CockpitManager is gonna be refactored.